functions.c

```diff
@@ -135,26 +136,86 @@ int unimode(int learning)
     }
 
     printf("You have left The ZEN ᕙ(⇀‸↼)ᕗ\n");
-    printf("To start The ZEN ᕙ(⇀‸↼)ᕗ again just type the command 'Unishell'.\n");
+    printf("To start The ZEN ᕙ(⇀‸↼)ᕗ again just type the command 'unimode'.\n");
   }
 
   return 1;
 }
 
-void pomodoro(int learn_int, int pause_int)
+void printWithTime(int h, int m, int s, int pause, int i, int interval)
 {
+  char cwd[PATH_MAX];
```

Comment:
If you want we can do a function for the console output, but since time is spare we can merge this